sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/namedExpressions.scala

```diff
@@ -155,6 +155,8 @@ case class Alias(child: Expression, name: String)(
         explicitMetadata == a.explicitMetadata
     case _ => false
   }
+
+  def cleanExprId(): Alias = Alias(child, name)(ExprId(-1), qualifiers)
```

Comment:
Why does this have `()`?  It also needs scala doc.  However, since its only ever used once, I'd consider just inlining it.